README.md

```diff
@@ -42,13 +42,15 @@ and changing some files. In case things go wrong or you change your
 mind, you'll use git to undo the changes, so you don't want other
 things in the way.
 
-1. Create a file in your project named `rax-docs`, and add the
-[content of the rax-docs
-script](https://github.com/IDPLAT/rax-docs/blob/master/rax-docs) to
-it. (GH Enterprise authentication means I can't make a simple wget
-command to get the script.)
+1. Download the main `rax-docs` script:
+    ```
+    wget -O rax-docs https://raw.githubusercontent.com/IDPLAT/rax-docs/master/rax-docs && \
+        chmod +x rax-docs
+    ````
 
-1. Install it. Find the [release](https://github.com/IDPLAT/rax-docs/releases) you want and use that as the version to install:
+1. Install the Toolkit. Find the
+[release](https://github.com/IDPLAT/rax-docs/releases) you want and
+use that as the version to install:
 
     ```
     chmod +x rax-docs
```

Comment:
No need for this anymore. You included the chmod in the instructions to download the script
```suggestion
```

Reply:
Oops. [Fixed](https://github.com/IDPLAT/rax-docs/compare/eaadeb93e8396c929db0b7504de32bb73e1a9c41..c24fea92b956807c851dd8227bfc6e79dfb74176). This will now conflict with #16.